FirebaseAI/Sources/Types/Internal/Live/LiveSessionService.swift

```diff
@@ -308,11 +308,12 @@ actor LiveSessionService {
   ///
   /// Will apply the required app check and auth headers, as the backend expects them.
   private nonisolated func createWebsocket() async throws -> AsyncWebSocket {
+    let host = apiConfig.service.endpoint.rawValue.withoutPrefix("https://")
```

Comment:
This is fine for now but I think we should consider updating `APIConfig.Service.Endpoint` (in a future PR) to store just the hostname and adding `https://` or `wss://` when it's used instead of trimming it.